orif/plafor/Database/Migrations/2024-07-04-082025_AddGrade.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Migration file for the table "grade"
+ *
+ * @author      Orif (ViDi, ThJo)
+ * @link        https://github.com/OrifInformatique
+ * @copyright   Copyright (c), Orif (https://www.orif.ch)
+ */
+namespace Plafor\Database\Migrations;
+
+use CodeIgniter\Database\Migration;
+
+class AddGrade extends Migration{
+
+    public function up(){
+        $this->db->disableForeignKeyChecks();
+        $this->forge->addField([
+            "id" => [
+                "type" => "INT",
+                "constraint" => "11",
+                "unsigned" => true,
+                "auto_increment" => true,
+            ],
+            "fk_user_course" => [
+                "type" => "INT",
+                "null" => true,
+                "unsigned" => true,
+            ],
+            "fk_teaching_subject" => [
+                "type" => "INT",
+                "null" => true,
+                "unsigned" => true,
+            ],
+            "fk_teaching_module" => [
+                "type" => "INT",
+                "null" => true,
+                "unsigned" => true,
+            ],
+            "date" => [
```

Comment:
La syntaxe is incorrecte. Cela ne fait pas d’erreur lorsqu’on lance `php sparke migrate` mais la colonne n’est pas créée.

Reply:
C'est fait: 
`            "date" => [
                "type" => "date",
            ],`